jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  roots: ['<rootDir>/api-test'],
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  testRegex: '(/__tests__/.*|(\\.|/)([i]?test|[i]?spec))\\.[jt]sx?$',
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
+};
```

Comment:
We store all our jest config here: https://github.com/coordinape/coordinape/blob/7231ec28b8d53090fd58d386698ad0aa8b213be0/craco.config.js#L25-L58

and run test via `yarn craco test`, so this can be deleted. Sorry this wasn't communicated before.

Reply:
Ah, should have searched for it better, thanks for pointing this out. Removed with https://github.com/coordinape/coordinape/pull/1486/commits/00dd7426ef469c69cbc2ed4514a9cf9bb065ffe4

Reply:
@levity removing this file caused the SyntaxError:

> SyntaxError: Cannot use import statement outside a module

Unsure if we can use this craco config for our Jest setup in ci @topocount, perhaps need to bring the file back

Reply:
@karelianpie just FYI this can be fixed by uncommenting this line: 
https://github.com/karelianpie/coordinape/blob/438e6178130f7d4db490d7cb542df4d90b0e0901/scripts/ci.sh#L62-L67
and deleting the explicit `jest` invocation at the bottom of the file.